app/views/government/collections/create-or-update-a-local-plan.html

```diff
@@ -1436,15 +1439,17 @@ <h2 class="gem-c-contents-list__title">Contents</h2>
                     </ul>
                   </div>
                 </div>
-                <div class="responsive-bottom-margin">
-                  <div class="app-c-published-dates" lang="en">
-                    Published 18 December 2023
+                <div class="published-dates-button-group">
```

Comment:
Added this wrapping div with the published-dates-button-group because that's what we're using to find the bottom of the content and this class ensures that the padding is included in the height of this div (otherwise the actual bottom of the content would be slightly further down that we calculate).